Dockerfile

```diff
@@ -468,6 +468,11 @@ ENV GOCACHE "/opt/buildhome/.gimme_cache/gocache"
 # Install the default version
 ENV GIMME_GO_VERSION "1.19.x"
 ENV GIMME_ENV_PREFIX "/opt/buildhome/.gimme/env"
+ENV GIMME_VERSION_PREFIX "/opt/buildhome/.gimme/versions"
+# Default is auto which on invalid go versions starts cloning git repos and searching through git and tries to build from git
+ENV GIMME_TYPE "binary"
```

Comment:
This will not allow installs from git anymore. Previously one could specify a git commit and gimme would clone the repo and start building.
When we do `gimme --resolve something` with an invalid version it will also clone the go repo and start searching if there is a commit that matches.